### DISCUSSION TITLE
Fail to upload institution proof

### DISCUSSION THREAD

**Discussion by cwu116** [community/community#55488] [GitHub Education]
### Select Topic Area

Question

### Body

I upload my student id and my school info, but github failed to give me the student authentication. When I want to upload again, it said I can't use pics which are used before

**Comment by samus-aran** [comment]
Hi @cwu116 

Yes, we do ask you take another image if the system detects that the same image is used. It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
